src/inference/src/os/common_table_op.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_table_op.hpp"
+
+#include "openvino/runtime/system_conf.hpp"
+
+namespace ov {
+
+void update_table_for_proc(const int _processor_id,
```

Comment:
Why we need to exposure this function? Do you have expectation developers will be needed to call it explicitly?
I would propose to keep it as private method of CPU class.
Tests should have its own implementation of this utility.

Reply:
Move this function into CPU class. Now this is not private method because specific test case for this function. 

@dmitry-gorokhov is it ok?

Reply:
As I mentioned test should have own implementation of this utility. I don't think this is good practice to exposure some functionality as dev API only in order to access it in the tests.